### PR TITLE
One split per line for MOOSE inputs.

### PR DESCRIPTION
**Pull request by elementx54**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
idaholab/raven#792

##### What are the significant changes in functionality due to this change request?
The MOOSE input line is now only split once with the left most `=` being
used.

Closes idaholab/raven#792

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?

### CODE REVIEW

**Comment by PaulTalbot-INL** [comment]
This PR needs a new test (or extension of an existing test) that demonstrates capturing the new behavior.

**Comment by elementx54** [comment]
There is no test to check MOOSE input files for parsing them correctly, so I couldn't modify an existing test to include multiple equal signs.

**Comment by elementx54** [comment]
Edit: Added the interface check that didn't exist and made sure the multiple equal signs problem was in the input file.  

Should be ready to merge (if it passes).

File: tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/2/out~MOOSE_parser [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (alfoa) [inline tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/2/out~MOOSE_parser]: this file should be removed

File: tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/1/out~MOOSE_parser.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+time,average_fuel_temperature,average_graphite_temperature,delta_temp,total_heat_source
+0,0,0,0,0
+0.001,264.95985083426,202.02485982188,62.934991012381,5540.0527538609
+0.002,265.15753682374,202.17015011614,62.987386707599,5545.5872720965
```
Comment (alfoa) [inline tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/1/out~MOOSE_parser.csv]: I do not recall. Is this file generated by the Code Interface or by MOOSE itself? (In the first case, it should be removed)

File: tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/1/out~MOOSE_parser [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (alfoa) [inline tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/1/out~MOOSE_parser]: this should be removed

File: tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/2/out~MOOSE_parser.csv [hunk 4]
```diff
@@ -0,0 +1,2 @@
+time,average_fuel_temperature,average_graphite_temperature,delta_temp,total_heat_source
+0,0,0,0,0
```
Comment (alfoa) [inline tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/2/out~MOOSE_parser.csv]: I do not recall. Is this file generated by the Code Interface or by MOOSE itself? (In the first case, it should be removed)

Comment (PaulTalbot-INL) [reply tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/2/out~MOOSE_parser.csv]: I agree the out~MOOSE_parser should be removed, but isn't out~MOOSE_parser.csv one of the output files that the interface check scoops up and checks collecting output for?

Comment (alfoa) [reply tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/2/out~MOOSE_parser.csv]: Sorry, I forgot.
Yes the CSVs should stay.

File: tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/2/out~MOOSE_parser (hunk 1 above)
Comment (elementx54) [reply tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/2/out~MOOSE_parser]: Removed!

File: tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/1/out~MOOSE_parser (hunk 3 above)
Comment (elementx54) [reply tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/1/out~MOOSE_parser]: Also removed.

File: tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/2/out~MOOSE_parser.csv (hunk 4 above)
Comment (elementx54) [reply tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/2/out~MOOSE_parser.csv]: Yep, the interface check needs it to pick up stuff after it *pretends* to run the executable. I left it in the amended commit.

**Comment by elementx54** [comment]
I've amended the commit with the requested changes (the two files are gone), and it is ready to restart the tests.

**Comment by elementx54** [comment]
So the OpenSUSE Tumbleweed test failed because it was never ran. Am I reading that correctly?

**Comment by alfoa** [comment]
yes

**Comment by elementx54** [comment]
Ah, so can you re-run that one since it looks like that is the only thing holding up this request? I don't think I can tell it to re-run.

**Comment by alfoa** [comment]
Checklist passed...

Merging...

**Pull request merged**